Hunting Queries/MultipleDataSources/StorageAlertCorrelationwithCommonSecurityLogsandStorageLogs.yaml

```diff
@@ -0,0 +1,48 @@
+id: 7098cae1-c632-4b40-b715-86d6b07720d7
+name: Storage Alert Correlation with CommonSecurityLogs and StorageLogs
+description: |
+  'This query combines different Storage alerts with CommonSecurityLogs and StorageLogs helping analysts  triage and investigate any 
+  possible Storage related attacks faster thus reducing MTTR'
+requiredDataConnectors:
+  - connectorId: Fortinet
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert (ASC)
```

Comment:
SecurityAlert as with the others

Reply:
Looks like this is the valid data type for ASC alerts  as per Wiki -https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#datatypes